specification/azsadmin/resource-manager/update/readme.md

```diff
@@ -65,14 +77,18 @@ This block is updated by an automatic script. Edits may be lost!
 
 ``` yaml
 # include the azure profile definitions from the standard location
-require: $(this-folder)/../../../../profiles/readme.md
+require: $(this-folder)/../../../../../profiles/readme.md
```

Comment:
why you change this folder , it seems the previous version is correct

Reply:
I remember it was showing a error saying the file couldn't be found under 
"/specification/profiles/readme.md"
after changing like this the error is gone.